sys/src/9/boot/boot.c

```diff
@@ -298,15 +298,15 @@ static void catstuff(void)
 	ifd = open("#t/eia0", ORDWR);
 	if (ifd < 0) {
 		print("can't open eia0!!!\n");
+		exits("can't open eia0");
 		return;
 	}
 	print("open: ifd %d\n", ifd);
 	while (1) {
 		sleep(100000);
 	}
-
-	print("all done!\n");
 }
```

Comment:
was there a reason to remove the all done print? just wondering.

Reply:
It'll never be reached, so why keep it?

Reply:
What I really mean is - if this could be reached, we should have an exits() call after it.  If it can't be reached, it should be a comment at best (IMO).

Reply:
Anyway, thanks for the review.